general-configuration-guide/basic-setup.md

```diff
@@ -63,7 +62,7 @@ tinymce.init({
 Providing a {{site.productname}} editor with the default configuration, such as:
 {% include codepen.html id="default" %}
 
-This selector can be used on any block element while using this option in {{site.productname}}'s [inline editing mode](../use-tinymce-inline/). Inline editing mode edits the content in place instead of replacing the element with an `iframe`.
+The selector can target any block element when the editor is used in [inline editing mode](../use-tinymce-inline/). Inline mode edits the content in place, instead of replacing the element with an `iframe`.
```

Comment:
This isn't actually true. TinyMCE will no initialize on certain block elements:
```
    invalidInlineTargets = Tools.makeMap(
      'area base basefont br col frame hr img input isindex link meta param embed source wbr track ' +
      'colgroup option table tbody tfoot thead tr th td script noscript style textarea video audio iframe object menu',
      ' '
    );
```

Reply:
now: `can target most block elements`

Reply:
And added the list to https://www.tiny.cloud/docs/configure/editor-appearance/#inline